ci/Dockerfile

```diff
@@ -15,24 +102,22 @@ RUN export DEBIAN_FRONTEND=noninteractive \
  && sudo add-apt-repository -y ppa:git-core/ppa \
  && sudo apt update -yq \
  && sudo apt install -yq --fix-missing \
-      build-essential cmake git s3cmd libpcre3-dev libnss3 libxcomposite1 libxtst6 jq gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly gstreamer1.0-libav gstreamer1.0-tools gstreamer1.0-alsa gstreamer1.0-pulseaudio
+      build-essential cmake git s3cmd libpcre3-dev libnss3 libxcomposite1 libxtst6 jq \
+      gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly \ 
+      gstreamer1.0-libav gstreamer1.0-tools gstreamer1.0-alsa libpulse-mainloop-glib0 \
+      gstreamer1.0-pulseaudio libxrandr2 libxcursor1 libxi6 libxcb-icccm4 libxcb-image0 \
+      libxcb-keysyms1 libxcb-render-util0 libxcb-render0 libxcb-xinerama0
```

Comment:
Wouldn't it make sense to combine all the `apt install commands` into one for one less layer?
Just separate dependencies with comments to indicate what is what.

Reply:
If we split the build into two separate images this comment is irrelevant.